securedrop_client/api_jobs/downloads.py

```diff
@@ -181,21 +180,17 @@ def _decrypt(self,
             original_filename = self.call_decrypt(filepath, session)
             db_object.download_error = None
             mark_as_decrypted(
-                type(db_object), db_object.uuid, session, original_filename=original_filename
-            )
-            logger.info("File decrypted: {} (decrypted file in: {})".format(
-                os.path.basename(filepath), os.path.dirname(filepath))
-            )
+                type(db_object), db_object.uuid, session, original_filename=original_filename)
+            logger.info(f'File decrypted to {filepath}')
```

Comment:
I think here in the decrypt method, on success we should provide the enclosing directory (i.e. log that the file can be found in `os.path.dirname(filepath)`), since the file at `filepath` will not exist if the user is looking for it - what do you think?